### ISSUE TITLE
Add a way to manage error on deletion

### ISSUE THREAD

**Issue by Kmaschta** [marmelab/ng-admin#980]
Let's consider this configuration:
```js
admin.errorMessage(function (response) {
    console.error(response);
    return response.data.error || JSON.stringify(response);
});

nga.configure(admin);
```
If an error is threw on deletion, it is not intercepted by `errorMessage` so response body is displayed to the user.

**Comment by fzaninotto** [comment]
Fixed by #982

**Comment by yangBin666** [comment]
[UPGRADE-1.0.md](https://github.com/marmelab/ng-admin/blob/defaa6ca31afe24d7f2fab1530fe5e574c0cf6a9/UPGRADE-1.0.md)
